src/index.user.js

```diff
@@ -4547,25 +4547,30 @@ function hookUpDailyChallengeAutoCollect() {
     );
     claimAllButtonElement.style.float = "right";
     claimAllButtonElement.style.marginRight = "16px";
-    claimAllButtonElement.textContent = "Claim All";
+    claimAllButtonElement.textContent = "Claim All & Close";
     claimAllButtonElement.addEventListener("click", clickHandler);
     titleRootElement.appendChild(claimAllButtonElement);
 
     function clickHandler() {
       const claimButtonElements = document.querySelectorAll(
         ".missionWrapper:not(.alreadyClaimed) .claimButton, .rewardWrapper:not(.claimed) .claimButton"
       );
+      const closeBtn = document.querySelector('#dailyMissionsPopup > a.close.closeButton[title="Close"]');
       if (!claimButtonElements.length) {
         console.log("No more challenges to claim.");
+        closeBtn.click();
         return;
       }
       console.log(`Challenges to claim: ${claimButtonElements.length}`);
       for (let claimButtonElement of claimButtonElements) {
         claimButtonElement.click();
       }
+
+      const timeToWait = Math.floor(1500 + Math.random()*(2500 - 1500 + 1)); // Random int between 1500 and 2500
```

Comment:
It would be very benefitial to include units in a variable name. In our case `timeToWaitInMs`.

Reply:
will do